packages/yoastseo/src/scoring/assessments/seo/ProductSKUAssessment.js

```diff
@@ -55,6 +54,27 @@ export default class ProductSKUAssessment extends Assessment {
 		return assessmentResult;
 	}
 
+	/**
+	 * Contains extra logic for the isApplicable method.
+	 *
+	 * @param {object} customData The custom data part of the Paper object.
+	 * @private
```

Comment:
I am not sure if adding `@private` in the JSDoc is necessary in this case. At least we haven't done this before in other assessments classes that we have.